README.md

```diff
@@ -6,14 +6,18 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
-### `npm start`
+### `npm start && json-server --watch public/store/footer-contact.json --port 3030`
 
 Runs the app in the development mode.\
 Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
 The page will reload if you make edits.\
 You will also see any lint errors in the console.
 
+json-server --watch public/store/footer-contact.json --port 3030 
```

Comment:
Can I access json files just as a static resource via fetch function? Why do we need this at all?

Reply:
Not needed at this stage, I have removed this functionality and no longer use it. Readme cleaned